Jenkinsfile

```diff
@@ -23,6 +23,15 @@ pipeline {
   }
 
   stages {
+    stage('Pre-Deploy') {
+//      failFast true
+//      when {
+//        branch 'master'
+//      }
+      steps {
+        verifyPacticipants("selfservice", gitCommit(), "test")
```

Comment:
Not sure we should use their terminology - this will not be clear to your average dev. Maybe something like `checkPactCompatibility` or something?